src/components/bulkConcepts/component/CardBody.jsx

```diff
@@ -10,7 +10,11 @@ const CardBody = ({ title, body }) => (
 
 CardBody.propTypes = {
   title: PropTypes.string.isRequired,
-  body: PropTypes.string.isRequired,
+  body: PropTypes.string,
```

Comment:
Is `body` not required?

Reply:
When `Add CIEL Concepts` is being clicked, `CardBody` will be called before concepts are in the store. Making it required causes a warning since props passed to `CardBody` are obtained from the store.